hardware/fab/turbocan-NPTH.drl

```diff
@@ -1,17 +1,17 @@
 M48
-; DRILL file {KiCad (5.1.9-0-10_14)} date Wednesday, June 30, 2021 at 10:16:16 AM
-; FORMAT={-:-/ absolute / metric / decimal}
-; #@! TF.CreationDate,2021-06-30T10:16:16+02:00
-; #@! TF.GenerationSoftware,Kicad,Pcbnew,(5.1.9-0-10_14)
+; DRILL file {KiCad (5.1.10)-1} date 07/05/21 19:11:22
+; FORMAT={-:-/ absolute / inch / decimal}
```

Comment:
Please use metric units for the drill files

Reply:
Hah, oops. I think I saw that the plot and footprint generation dialog had millimeters set as and assumed everything else would default to metric as well. Seems JLC handles mixed units though, it must have been set that way when I ordered my boards.

Should be fixed in next commit.